styles.css

```
@@ -1,6 +1,40 @@
 @charset 'UTF-8';
 
-h1 {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 2.4rem;
+  font-size: 1.2rem;
   font-family: Arial, Helvetica, sans-serif;
-  color: blueviolet;
+  background-color: #f3f3f3;
+}
+
+nav a {
+  text-decoration: none;
+}
+
+a:link {
+  color: black;
+}
+
+a:visited {
+  color: black;
+}
+
+img.logo {
+  width: 2.5rem;
+}
+
+ul {
```

```suggestion
nav ul {
```
Quando voce define o estilo usando apenas o elemento `ul` esse estilo vai ser aplicado para qualquer outra listas que criarmos. Por isso é interessante definir que esse estilo é apenas para a `ul` que pertence ao `nav`.
Seria interessante criarmos classes ou id's para os nossos elementos html, assim vai ficar mais facil nao nos perdermos quando o codigo ficar maior (podemos discutir isso na nossa proxima reuniao).